amundsen_application/api/jira/v0.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from http import HTTPStatus
+from flask import Response, jsonify, make_response, request
+from flask.blueprints import Blueprint
+from flask import current_app as app
+from typing import Any
+
+from amundsen_application.jira.jira_client import JiraClient
+from amundsen_application.api.utils.request_utils import get_query_param
+
+LOGGER = logging.getLogger(__name__)
+
+jira_blueprint = Blueprint('jira', __name__, url_prefix='/api/jira/v0')
+
+
+@jira_blueprint.route('/issues', methods=['GET'])
+def get_jira_issues() -> Response:
+    """
+    Given a table key, returns all JIRA tickets containing that key. Returns an empty array if none exist
+    :return: List of JIRA tickets
+    """
+    try:
+        table_key = get_query_param(request.args, 'key', 'Request requires a key')
+        missing_config_response = validate_jira_properties()
```

Comment:
This seems like something JIRA client should validate, or should be validated while JIRA client is being instantiated. WDYT?

Reply:
definitely agree - i had moved it out because it seemed like a weird pattern to raise an exception and return an error message instead.